### PR TITLE
Support for Symfony 4

### PR DESCRIPTION
**Pull request by Uneo7**
I've edited a few things for this bundle to work correctly with Sf 4 and updated the documentation consequently.

### CODE REVIEW

File: Resources/doc/installation.md [hunk 1]
```diff
@@ -3,12 +3,25 @@ Installation
...
+```bash
+php composer.phar require templating
+```
+
+and configure it with your favorite engine : 
+
+```yaml
+framework:
+    templating:
+        engines: twig
```
Comment (cordoval) [inline Resources/doc/installation.md]: can we avoid this?

Comment (Uneo7) [reply Resources/doc/installation.md]: Seems no unfortunately. The templating service used to render the form won't work without it.

Comment (cordoval) [reply Resources/doc/installation.md]: are you forced to use the form?

Comment (Uneo7) [reply Resources/doc/installation.md]: Yes it's the form used to enter your TOTP code.

**Comment by Uneo7** [comment]
@scheb

**Comment by scheb** [comment]
Looks good to me, thanks for the effort!

File: Resources/doc/configuration.md [hunk 2]
```diff
@@ -3,8 +3,10 @@ Configuration
 
 For detailed information see the documentation of the authentication methods, [Google Auth](google.md) and [Email](email.md).
 
+With Symfony >= 4 we must add configuration file ourself : /config/packages/scheb_two_factor.yaml
```
Comment (cordoval) [inline Resources/doc/configuration.md]: typo ourselves

File: Resources/doc/installation.md (hunk 1 above)
Comment (cordoval) [reply Resources/doc/installation.md]: well that is only if you use it right?

File: Resources/doc/configuration.md (hunk 2 above)
Comment (scheb) [reply Resources/doc/configuration.md]: Fixed, thanks!

File: Resources/doc/installation.md (hunk 1 above)
Comment (Uneo7) [reply Resources/doc/installation.md]: Indeed you're right i've forgetten the mail feature ^^ I'll move this part in the Google Authenticator's doc.

Comment (cordoval) [reply Resources/doc/installation.md]: i am glad please fix it asap 👍

**Pull request merged**